services/portal/GWASpp/GWASService.js

```diff
@@ -0,0 +1,11 @@
+const GWASppTasks = require('../GWASpp/GWASppTasks.js');
+const GWASppProps = require('../GWASpp/GWASppProps.js');
+
+/**
+ * fence Service
+ */
+module.exports = {
+    props: GWASppProps,
+
+    do: GWASppTasks,    
+};
```

Comment:
I believe github is looking for a newline at the end of this file. You should be able to just add an extra line with a return and github will be happy. 

Here's an explanation for why github does this: https://gist.github.com/camh-/1bebfcff1b0f814e9b191edc60d5206b